.env.example

```diff
@@ -1,7 +1,27 @@
 # Example .env file for testing
 
+ENVIRONMENT=development
+DEBUG=true
 SECRET_KEY=secret
-ALLOWED_HOSTS=[]
+
+ALLOWED_HOSTS=localhost,127.0.0.1
+SITE_ID=4
+
+# Database
+DB_USER=physionet
+DB_PASSWORD=password
+DB_HOST=localhost
+DB_NAME=physionet
+
+# Emails
+EMAIL_HOST=localhost
+DEFAULT_FROM_EMAIL=localhost@localhost
+CONTACT_EMAIL=PhysioNet Contact <contact@dev.physionet.org>
```

Comment:
not sure whether this changes anything functionally, but perhaps good for use of quotes to be consistent with the line below.

Reply:
Fixed!